tests/plugins/test_plugins_manager_www.py

```diff
@@ -83,3 +85,66 @@ def test_menu_links(self):
         [menu_link] = [ml for ml in category.get_children()
                        if isinstance(ml, MenuLink)]
         self.assertEqual('Test Menu Link', menu_link.name)
+
+
+class MockPluginA(AirflowPlugin):
```

Comment:
sorry, but could you move the MockPlugin to test_plugin file which is a centralized place for all the test plugins?

Reply:
Done, I just have to wait for the tests to run again